### PR TITLE
feat: support for existing event notification instance

### PR DESCRIPTION
**Pull request by iamar7**
### Description

Resolves: https://github.com/terraform-ibm-modules/stack-ibm-core-security-services/issues/50

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: tests/pr_test.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func TestProjectsFullTest(t *testing.T) {
 		"prefix":                            options.Prefix,
 		"region":                            validRegions[rand.Intn(len(validRegions))],
 		"existing_resource_group_name":      resourceGroup,
-		"sm_service_plan":                   "trial",
+		"sm_service_plan":                   "standard",
```
Comment (ocofaigh) [inline tests/pr_test.go:59]: why are using standard? That very expensive to use for every PR test. We have an extended quota of allowed trial instances in our account, so please set this back to trial

Comment (iamar7) [reply tests/pr_test.go:59]: Was getting `This account reached the number of Trial plan instances that are allowed` for number of pipeline run due to which updated it to `standard`.

**Review by akocbek — APPROVED** [review]

**Pull request merged**